core/resources/l10n/messages.properties

```diff
@@ -767,6 +767,8 @@ simplotpanel.but.NewYaxisplottype = New Y axis plot type
 simplotpanel.lbl.Axis = Axis:
 simplotpanel.but.ttip.Deletethisplot = Delete this plot
 simplotpanel.Desc = The data will be plotted in time order even if the X axis type is not time.
+! Fix this name
+simplotpanel.Warning = The data is plotted in time order even though the X axis type is not time. 
```

Comment:
You'd have to excuse me, my initial comment to put the new translation key was not very good. It's better to look for the keys `PlotDialog` and add a new key, for instance `PlotDialog.lbl.timeSeriesWarning`